lib/client_summarizer.rb

```diff
@@ -0,0 +1,31 @@
+class ClientSummarizer
+  include FiscalYearMixin
+
+  attr_reader :client_namespace, :fiscal_year
+
+  def initialize(args)
+    @client_namespace = args[:client_namespace]
+    now = Time.zone.now
+    @fiscal_year = (args[:fiscal_year] || self.class.which_fiscal_year(now.year, now.month)).to_i
+    @_records = build_records
```

Comment:
interesting. I typically use underscore more generally to indicate the "privateness" of a variable. I realize there's no attr_reader for records, so it's already "private" in that sense. Perl habit I guess (where there is no sense of private/public for methods except by convention).

Is underscore-for-memoization-onl a more common Ruby convention?